scripts/check-asciidoctor-build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# ensure asciidoctor is installed
+if ! command -v asciidoctor ;
+then
+    echo "Asciidoctor is not installed. Please install it and try again."
+    exit 127
+fi
+
+# get the *.adoc modules and assemblies in the pull request
+FILES=$(git diff-tree HEAD HEAD~1 --no-commit-id --name-only -r)
+
+# get the modules in the PR, search for assemblies that include them, and concat with any updated assemblies files
+get_updated_assemblies () {
+    MODULES=$(echo "$FILES" | awk '/modules\/(.*)\.adoc/')
```

Comment:
**thought:** Are we using this?

Reply:
It's used in https://github.com/openshift/openshift-docs/pull/60116/files#diff-27d906f6adff2ae2ddd47496ea2cb19b2b4a5cac0f03fd3f280f11f6a7a55d63R19

If we don't do this we search for updated assemblies in assemblies too.